webapp/components/User/User.vue

```diff
@@ -14,14 +14,15 @@
             <b class="username">{{ userName | truncate(18) }}</b>
           </div>
           <!-- Time -->
-          <div v-if="dateTime" style="display: inline;">
-            <ds-text align="left" size="small" color="soft">
+          <ds-text align="right" size="small" color="soft">
+            <span class="slug">{{ userSlug }}</span>
+            <div v-if="dateTime" style="display: inline;">
```

Comment:
If you use a `<span>` here instead of the `<div>` there's no need for the `inline` style. It also makes more sense semantically. ;)